android/src/main/kotlin/com/abedalkareem/games_services/GamesServicesPlugin.kt

```diff
@@ -67,6 +85,9 @@ class GamesServicesPlugin(private var activity: Activity? = null) : FlutterPlugi
 
     //region Leaderboards
     private fun showLeaderboards(result: Result) {
+        if (leaderboardsClient?.allLeaderboardsIntent == null) {
+            Toast.makeText(activity, "Please log to Google Play Games", Toast.LENGTH_LONG).show();
```

Comment:
I don't believe that we should show this error here, the developer should have the ability to display a custom flutter dialog.

Reply:
Certainly & sorry I should have a new branch to experiment on.

Reply:
Sorry for the late reply @Abedalkareem . I did this because on iOS attempt to show a leaderboard iOS is displaying a similar behaviour. So in order to make the experience consistent I added the same message. If the developer want to replace but his custom message he can always do that